### PR TITLE
feat(testing) add rtl tests for components

### PR DESCRIPTION
**Pull request by yiyang-qantashotels**
## Description

- Container
- CollapsibleBanner
- CalendarDay
- Checkbox

💁‍

## Related issues

💪

## Looks like this

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan), [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: src/components/Container/Container.jsx [hunk 1]
```diff
@@ -5,6 +5,10 @@ import omitProps from '../omitProps';
 
 const gutter = (props) => {
   const gutterValue = themeGet(`gutters.${props.gutter}`, props.gutter)(props);
+  console.log(
+    '🚀 ~ file: Container.jsx ~ line 8 ~ gutter ~ gutterValue',
+    gutterValue,
+  );
```
Comment (ajlu) [inline src/components/Container/Container.jsx:11]: Remove console.log?

File: src/components/DatePicker/components/CalendarDay/CalendarDay.test.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```
Comment (ajlu) [inline src/components/DatePicker/components/CalendarDay/CalendarDay.test.jsx:1]: Delete the snapshots and enzyme test for `CalendarDay`?

File: src/components/Container/Container.jsx (hunk 1 above)
Comment (yiyang-qantashotels) [reply src/components/Container/Container.jsx:11]: 🤦‍♂️ yeah, thanks for picking it up!

File: src/components/DatePicker/components/CalendarDay/CalendarDay.test.jsx (hunk 2 above)
Comment (ajlu) [reply src/components/DatePicker/components/CalendarDay/CalendarDay.test.jsx:1]: @yiyang-qantashotels also delete the enzyme test - https://github.com/hooroo/roo-ui/blob/master/src/components/DatePicker/components/CalendarDay/CalendarDay.test.js?

**Review by ajlu — APPROVED** [review]
Nicely done 👍 Just one minor comment

**Pull request merged**